### ISSUE TITLE
Database backup

### ISSUE THREAD

**Issue by Jordybastien** [rubynor/reap#115]
May not be needed for now but later on for sure, we can just set up a way to pull online DB locally for now

**Comment by str1fe** [comment]
Good point. As soon as we start adding real data here, we need to run backups

**Comment by mrtnin** [comment]
https://github.com/rubynor/backy
